README.md

```diff
@@ -50,15 +62,17 @@ Each intention can appear at any of the 4 risk levels. Each intention's full det
 
 | Code | Known Approaches |
 | --- | --- |
-| `f - ` | None known |
-| `F - ` | Meets all of:<ul><li>Change is <= 8 <abbr title="lines of code">LoC</abbr><sup>[5]</sup></li><li>Feature was fully unit tested prior to this change.</li><li>Change includes new or changed unit tests to match intended behavior alteration.</li></ul> |
-| `F!!` | Change includes unit tests for new behavior. |
-| `F**` | No automatic tests, or unfinished implementation. |
+| `. F` | Meets all criteria for `- F` and developers are the only users of the feature. For example, extends build tooling for your own build or adds debug logging. |
```

Comment:
In this version, I'm not seeing a `- F`. Does this refer to the requirements of `^ F`?